Sources/FoundationEssentials/FileManager/FileOperations.swift

```diff
@@ -988,11 +988,11 @@ enum _FileOperations {
             #endif
             
             // Copy modification date
-            let value = timeval(tv_sec: statInfo.st_mtim.tv_sec, tv_usec: statInfo.st_mtim.tv_nsec / 1000)
+            let value = statInfo.st_mtim
             var tv = (value, value)
             try withUnsafePointer(to: &tv) {
-                try $0.withMemoryRebound(to: timeval.self, capacity: 2) {
-                    if futimes(dstFD, $0) != 0 {
+                try $0.withMemoryRebound(to: timespec.self, capacity: 2) {
+                    if futimens(dstFD, $0) != 0 {
```

Comment:
Thanks, I just spent some time looking into and [working around this issue for Android](https://github.com/finagolfin/swift-android-sdk/pull/193/commits/acb41774f6a2fa77873c73648cca181ee04b089f#diff-1506819823642d2cff75731eef95eae7ce56408b2796cba3b55ee9a7749fbeccR9) before I saw your pull, so I'm all for this `futimens()` change.